dss/xraytrace.py

```diff
@@ -0,0 +1,28 @@
+import os
+from functools import wraps
+import typing
+
+DSS_XRAY_TRACE = int(os.environ.get('DSS_XRAY_TRACE', '0')) > 0  # noqa
+
+
+def config_xray() -> None:
+    if DSS_XRAY_TRACE:  # noqa
+        from aws_xray_sdk.core import xray_recorder, patch_all
+        patch_all()
+        xray_recorder.configure(context_missing='LOG_ERROR')
```

Comment:
Do you need to set the context here?

Reply:
I thought we only needed that when we weren't using the lambda_context within a lambda.